app/components/pages/StaticPages/ScrollToTop.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+
+class ScrollToTop extends React.Component {
```

Comment:
A component like this could use `React.PureComponent` or even the new `React.memo` from `v16.6.0` for improved performance.